client/web/antrea-ui/src/utils/fakeData.ts

```diff
@@ -0,0 +1,94 @@
+import { AgentInfo } from '../api/info';
+
+export function generateFakeAgents(n: number): AgentInfo[] {
+    // Helper function to generate random agent names
+    const generateAgentName = () => {
+        const prefixes = ['antrea', 'k8s', 'node', 'worker', 'master'];
+        const suffixes = ['-agent', '-node', '-vm', '-host', ''];
+        const prefix = prefixes[Math.floor(Math.random() * prefixes.length)];
+        const suffix = suffixes[Math.floor(Math.random() * suffixes.length)];
+        const id = Math.random() < 0.3 ? 
+            Math.random().toString(36).substring(2, 6) : // alphanumeric
+            Math.floor(Math.random() * 1000).toString(); // numeric
+        return `${prefix}${suffix}-${id}`;
```

Comment:
I don't understand the value of having such "complicated" code for generating agent names?
why not simply use `k8s-node-control-plane-1`, `k8s-node-worker-1`, `k8s-node-worker-2`, ...?